### ISSUE TITLE
Debian installation error

### ISSUE THREAD

**Issue by guidtz** [kaltura/platform-install-packages#639]
- [ ] I've read the [guidelines for Contributing to Kaltura Projects](https://github.com/kaltura/platform-install-packages/blob/master/CONTRIBUTING.md)
- [ ] I've read the [installation guides](https://github.com/kaltura/platform-install-packages#documentation-and-guides) and [troubleshooting FAQ](https://github.com/kaltura/platform-install-packages/blob/master/doc/kaltura-packages-faq.md)
- [ ] This is not a support request or question that should be posted on the [Kaltura forum](https://forum.kaltura.org) 

## Bug report

**Please provide steps to reproduce, including errors from /opt/kaltura/log/kaltura_api_v3.log, /opt/kaltura/log/kaltura_apache_errors*log and /opt/kaltura/log/kaltura_prod.log:**

**Linux distribution name and version:**
Debian 8

**Are you using the RPM or deb packages?**
- [ ] deb

**For deb based systems:**
```
# dpkg -l kaltura-*
Souhait=inconnU/Installé/suppRimé/Purgé/H=à garder
| État=Non/Installé/fichier-Config/dépaqUeté/échec-conFig/H=semi-installé/W=attend-traitement-déclenchements
|/ Err?=(aucune)/besoin Réinstallation (État,Err: majuscule=mauvais)
||/ Nom                                  Version                 Architecture            Description
+++-====================================-=======================-=======================-=============================================================================
ii  kaltura-base                         14.6.0-2                all                     Kaltura Open Source Video Platform
un  kaltura-php7-cli                     <aucune>                <aucune>                (aucune description n'est disponible)
un  kaltura-php7-curl                    <aucune>                <aucune>                (aucune description n'est disponible)
un  kaltura-php7-mcrypt                  <aucune>                <aucune>                (aucune description n'est disponible)
un  kaltura-php7-memcache                <aucune>                <aucune>                (aucune description n'est disponible)
un  kaltura-php7-mysql                   <aucune>                <aucune>                (aucune description n'est disponible)
ii  kaltura-postinst                     1.0.33-3+b9             all                     Kaltura Open Source Video Platform

```

**Is this an all in one instance [single server] or a cluster?**
- [ ] All in one

**If applicable, please provide the MySQL version**
ii  mysql-server-5.5               5.5.60-0+deb8u1 

**If you're having an issue with a specific media asset, please provide a link from which it can be downloaded/played:**

I have error when I try to install kaltura with install_kaltura_all_in_1.sh 

`Réception de : 25 http://installrepo.kaltura.org/repo/apt/debian/ naos/main kaltura-widgets all 14.1.0-1 [1 352 B]
14,5 Mo réceptionnés en 1s (7 738 ko/s)     
E: Impossible de récupérer http://installrepo.kaltura.org/repo/apt/debian/pool/main/k/kaltura-html5lib/kaltura-html5lib_2.71.5-1_all.deb  404  Not Found [IP : 143.204.192.12 80]

E: Impossible de récupérer http://installrepo.kaltura.org/repo/apt/debian/pool/main/k/kaltura-kmc/kaltura-kmc_5.43.13-8_all.deb  404  Not Found [IP : 143.204.192.12 80]
`
Sources list : 
`
# cat /etc/apt/sources.list.d/kaltura.list
deb [arch=amd64] http://installrepo.kaltura.org/repo/apt/debian naos main
`

Thanks for you're help

**Comment by jessp01** [comment]
Hi @guidtz,

We've just released a new version [14.7.0] and since we're using CDN, it may take a little while for all edges to sync with the origin server. If you're having issues with fetching http://installrepo.kaltura.org, please edit `/etc/apt/sources.list.d/kaltura.list` and replace 'installrepo.kaltura.org' with 'installrepo.origin.kaltura.org'. 

After that, run:
```sh
# apt-get update
```

Do let me know if you encounter additional issues. 

Thanks,

**Comment by guidtz** [comment]
Yes It's works with installrepo.origin.kaltura.org. I hava another error with kaltura database creation. I open another issue.
